robot_modeling/DQ_SerialManipulator.m

```diff
@@ -215,9 +129,35 @@ function set_effector(obj,effector)
             p(6) = h(2)*h(8)+h(6)*h(4)+h(1)*h(7)+h(5)*h(3);
             p(7) = h(3)*h(8)+h(7)*h(4)-h(1)*h(6)-h(5)*h(2);
             p(8) = h(4)*h(8)-h(3)*h(7)-h(2)*h(6)+h(1)*h(5);
+        end    
+
+        function x = raw_fkm(obj,q,to_ith_link)
+            %   RAW_FKM(q) calculates the forward kinematic model and
+            %   returns the dual quaternion corresponding to the
+            %   last joint (the displacements due to the base and the effector
+            %   are not taken into account).
+            %
+            %   'q' is the vector of joint variables
+            %   'to_ith_link' defines until which link the raw_fkm will be
```

Comment:
'to_ith_link' defines the last link that will be used in calculations of the forward kinematics.

Reply:
@bvadorno
Implemented in [bee728b](https://github.com/dqrobotics/matlab/pull/75/commits/bee728b9a09f90e7989f9f632b4e55d69cd2abfa).